src/Relation/Binary/Consequences.agda

```diff
@@ -44,6 +44,31 @@ module _ {_∼_ : Rel A ℓ} {P : Pred A p} where
   resp⇒¬-resp : Symmetric _∼_ → P Respects _∼_ → (∁ P) Respects _∼_
   resp⇒¬-resp sym resp x∼y ¬Px Py = ¬Px (resp (sym x∼y) Py)
 
+------------------------------------------------------------------------
+-- Proofs for negation
+
+module _ {_∼_ : Rel A ℓ} where
+  private
+    _≁_ : A → A → Set _
+    x ≁ y = ¬ (x ∼ y)
+
+  sym⇒≁-sym : Symmetric _∼_ → Symmetric _≁_
+  sym⇒≁-sym sym≁ x≁y y∼x = x≁y (sym≁ y∼x)
+
+  comp⇒≁-trans : Comparison _∼_ → Transitive _≁_
+  comp⇒≁-trans comp {i = x} {j = z} {k = y} x≁z z≁y x∼y = [ x≁z , z≁y ]′ (comp z x∼y)
```

Comment:
I think you can remove the implicit arguments here.

Reply:
In fact I kind of prefer keeping them because it highlights the difference in ordering between Comparison and Transitive (notice j <-> z and k <-> y). I can add a comment pointing that out if you think it could help?

Reply:
I guess I'd prefer a comment than the unused implicit arguments, and I think the comment should go on the main definition rather than here, but if you're keen to keep both then that's fine with me :+1:

Reply:
I've put comments in both places and removed the implicit arguments. Hopefully that's clear enough for future readers.